integration_tests/1pc_test.go

```diff
@@ -62,27 +61,26 @@ func (s *testOnePCSuite) TearDownTest() {
 }
 
 func (s *testOnePCSuite) Test1PC() {
-	ctx := context.WithValue(context.Background(), util.SessionID, uint64(1))
-
 	k1 := []byte("k1")
 	v1 := []byte("v1")
 
 	txn := s.begin1PC()
 	err := txn.Set(k1, v1)
 	s.Nil(err)
-	err = txn.Commit(ctx)
+	err = txn.Commit(context.Background())
 	s.Nil(err)
 	s.True(txn.GetCommitter().IsOnePC())
 	s.Equal(txn.GetCommitter().GetOnePCCommitTS(), txn.GetCommitter().GetCommitTS())
 	s.Greater(txn.GetCommitter().GetOnePCCommitTS(), txn.StartTS())
 	// ttlManager is not used for 1PC.
 	s.True(txn.GetCommitter().IsTTLUninitialized())
 
-	// 1PC doesn't work if sessionID == 0
+	// 1PC doesn't work if system variable not set
```

Comment:
I think we do not need this test anymore?

Reply:
We still need it. It's to check the default committing mode of a transaction. Now I change the comment to be more appropriate for this separated client.